Weekly Update 03-03-2017

nzoghb opened the pull request:
Late update as I was out of town. My 02-24-2017 update was not merged for some reason so I have added what I remember of it.

Commit:
Weekly Update 03-03-2017

Late update as I was out of town. My 02-24-2017 update was not merged
for some reason so I have added what I remember of it.

stefanv commented:
What do you mean your update was not merged?

stefanv commented on updates/Nick_Weekly_Updates.md:
```diff
@@ -0,0 +1,18 @@
+
+## 03-03-2017
+
+- **Past week:** Filed bug #2554 and worked on PR #2538. Scoured every file
```
Please provide links to the issues in question.

stefanv commented:
Hi Nick, thanks for working on the refactoring & deprecation; do you want to swing by today or tomorrow so that we can discuss?

Commit:
Minor Changes

URLs to all referenced Issues and PRs added to my updates.

Also added my name to the README; the names don’t seem to link
anywhere, will this be fixed?
